modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -1074,6 +1074,22 @@ func (h *HTTPTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 			}
 			h.ForwardProxyURL = d.Val()
 
+		case "forward_proxy":
```

Comment:
If it's "network_proxy" should that be the option name in Caddyfile?

Reply:
I was on the fence. I've changed them all to `network_proxy`.